### PR TITLE
WDR: add wdr4 and www1 subdomains

### PR DESCRIPTION
**Pull request by numismatika**
rest of the hosts from the SAN of the main certificate are omitted for now
There are small mixed content warnings but as far as i can tell the site doesn't break with the missing css.
Videos seem to work fine too.

### CODE REVIEW

**Comment by J0WI** [comment]
Could you also search for subdomains?

**Comment by numismatika** [comment]
There are a lot of domains in the cert for the mainpage, will take a while to see if they are even used on the web.

**Comment by J0WI** [comment]
You can [use Google](https://github.com/EFForg/https-everywhere/blob/master/ruleset-style.md) to search for subdomains, even those that have no or a different cert.

**Comment by J0WI** [comment]
Does this also address #2577 and #4227?

File: src/chrome/content/rules/Wdr.de.xml [hunk 1]
```diff
@@ -1,22 +1,79 @@
...
+		- ^wdr.de
 
-		- www1 ¹
+	certificatechain broken on:
+		- webmail.wdr.de
+		- freundeskreis.einslive.de
 
-	¹ Redirect to HTTP
+	Non-resolvable host:
+		- static.freundeskreis.einslive.de
```
Comment (J0WI) [inline src/chrome/content/rules/Wdr.de.xml]: If a host is not resolvable (=> does not exist) you don't need to list it.

File: src/chrome/content/rules/Wdr.de.xml [hunk 2]
```diff
@@ -1,22 +1,79 @@
...
+
+	Redirect to plain on:
```
Comment (J0WI) [inline src/chrome/content/rules/Wdr.de.xml]: `plain` is not accurate, please use 'http' or 'plain http'.

File: src/chrome/content/rules/Wdr.de.xml (hunk 1 above)
Comment (numismatika) [inline src/chrome/content/rules/Wdr.de.xml]: It is part of the SAN part of the certificate for www.wdr.de so for the sake of completeness i noted that

File: src/chrome/content/rules/Wdr.de.xml (hunk 2 above)
Comment (numismatika) [inline src/chrome/content/rules/Wdr.de.xml]: will be changed

**Comment by numismatika** [comment]
#2577 seems fine, videos on the sportschau page play. The live-stream placeholder loop is fine too.
Haven't found a real livestream yet for testing.
#4227 still is an issue. CSS&JS are blocked as mixed-content and FF ignores our rewrite (open FF bug)

**Comment by J0WI** [comment]
>  #4227 still is an issue. CSS&JS are blocked as mixed-content and FF ignores our rewrite (open FF bug)

So this rule needs either exclusions or spitted into a `platform="mixedcontent"` part.

**Comment by numismatika** [comment]
add `platform="mixedcontent"` to finally get this merged.
WDR takes too much time and not many things on the page are broken

**Comment by J0WI** [comment]
trigger Travis again

**Pull request merged**